.github/workflows/builds.yml

```diff
@@ -26,11 +26,11 @@ jobs:
             MSG_LVL: 5
         PLATFORM:
           - OS: "ubuntu-latest"
-            GENERATOR: "Unix Makefiles"
+            GENERATOR: -G "Unix Makefiles"
           - OS: "macos-latest"
-            GENERATOR: "Xcode"
+            GENERATOR: -G "Xcode"
```

Comment:
I think this is going away. 

Any reason for not using ninja on all platforms?

Reply:
I find that it tests the build configuration better if you use it with as many different generators as possible. A configuration that works with Ninja may very well not work with MSBuild, and vice versa.

On the `macos-latest` platform the "unique generator" available is Xcode. So I thought this would be a good way to make sure that this would continue working.

We could switch some of the builds to Ninja if you'd like. For instance all of the optimised container builds could be done with Ninja, while the debug builds would be done with GNU Make. 🤔

Reply:
Actually, scratch that. CMake apparently implemented support for Xcode 12+'s "new" build system. So let's stick to Xcode.